Library/Homebrew/formula.rb

```diff
@@ -1614,6 +1625,16 @@ class << self
     # @private
     attr_reader :plist_manual
 
+    # The block evaluated by `pour_bottle?` to decide whether to use a bottle
+    # or build from source.
+    # @private
+    attr_reader :pour_bottle_block
+
+    # If `pour_bottle?` returns `false` the user-visible reason to display for
+    # why they cannot use the bottle.
+    # @private
+    attr_reader :pour_bottle_reason
```

Comment:
Could we have a better name for this variable? Because it's actually not pouring reason.